src/HtmlUtils.tsx

```diff
@@ -645,7 +645,7 @@ export function bodyToHtml(content: IContent, highlights: Optional<string[]>, op
         />
     ) : (
         <span key="body" ref={opts.ref} className={className} dir="auto">
-            {emojiBodyElements || strippedBody}
+            {emojiBodyElements! || strippedBody}
```

Comment:
This looks wrong, if its always asserted (!) then why would you have a fallback to strippedBody?

Reply:
removed assertion